src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,21 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
-
-@_delegate_statsfuns Normal norm μ σ
-
 gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
+logpdf(d::Normal, x::Real) = -(x - d.μ)^2 / 2 / d.σ^2 - log(2π * d.σ^2) / 2
+pdf(d::Normal, x::Real) = exp(logpdf(d, x))
+cdf(d::Normal, x::Real) = (1 + erf((x - d.μ) / d.σ / √2)) / 2
```

Comment:
This will suffer from cancellation for small values (we should have a test for this). Look at how `normcdf` is implemented:
https://github.com/JuliaStats/StatsFuns.jl/blob/316cebb69d4e28e6a4b17d72eee112e61c3b9b77/src/distrs/norm.jl#L45

Reply:
also, you will need to define `ccdf` as well.

Reply:
This is not documented:
https://juliastats.github.io/Distributions.jl/stable/extends.html#Univariate-Distribution-1

Reply:
most newly implemented distributions haven't done it I think

Reply:
No, but if we don't we will get the same cancellation effects, since otherwise it will fallback on `1-cdf(d,x)`

Reply:
cc @Nosferican